cognite/client/utils/_time.py

```diff
@@ -46,6 +46,14 @@ def import_zoneinfo() -> ZoneInfo:
         ) from e
 
 
+def import_zoneinfo_error() -> ZoneInfoNotFoundError:
+    if sys.version_info >= (3, 9):
+        from zoneinfo import ZoneInfoNotFoundError
+    else:
+        from backports.zoneinfo import ZoneInfoNotFoundError
+    return ZoneInfoNotFoundError  # type: ignore [return-value]
```

Comment:
This is redundant since we do the same conditional import on l18-l21, right?

Reply:
No, this is the import at runtime, wile the l18-l21 is only type checking as far as I understand it.